File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { IpIntelligenceApi2SDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await IpIntelligenceApi2SDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- IpIntelligenceApi2 SDK exists test

local sdk = require("ip-intelligence-api2_sdk")

describe("IpIntelligenceApi2SDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
